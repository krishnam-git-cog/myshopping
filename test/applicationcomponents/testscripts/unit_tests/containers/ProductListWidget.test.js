import React from 'react';
import { createShallow } from '@material-ui/core/test-utils';
import ProductListWidget from '../../../../../src/containers/plp/ProductListWidget';

describe('ProductListWidget test', () => {
    it("Container - ProductListWidget should match snapshot", () => {
        const match = { params: { id: 'foobar' } }
        let shallow = createShallow();
        const component = shallow(<ProductListWidget match={match}/>);
        expect(component).toMatchSnapshot();
    });
});