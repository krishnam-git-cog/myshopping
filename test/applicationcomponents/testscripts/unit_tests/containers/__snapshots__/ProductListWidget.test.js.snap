// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductListWidget test Container - ProductListWidget should match snapshot 1`] = `ShallowWrapper {}`;
